### DISCUSSION TITLE
Release 1.0.0

### DISCUSSION THREAD

**Discussion by IsaacMarovitz** [Whisky-App/Whisky#189] [Announcements]
**Quality of Life:**
- Added toggle for Retina Mode
- Renamed GPT to GPTK to match community lingo
- Easy play button added to Start Menu items
- Windows paths are used where possible
- Ventura users are now alerted of potential problems
- Logging improvements

**New Features:**
- Libraries are now downloaded, not bundled
- Rosetta 2 Installer
- Programs can now be favourited

**Bug Fixes:**
- Removed duplicate Start Menu entries
- Several fixes and UI improvements to in-flight bottle handling
- Fixed a crash related to icon extraction
- Fixed stuck build number spinner

**New Contributors:**
* @JoshuaBrest made their first contribution in https://github.com/IsaacMarovitz/Whisky/pull/152
* @benwaco made their first contribution in https://github.com/IsaacMarovitz/Whisky/pull/161

**Full Changelog**: https://github.com/IsaacMarovitz/Whisky/commits/v1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/IsaacMarovitz/Whisky/releases/tag/v1.0.0'>Release 1.0.0</a>.</em>

**Comment by logiota** [comment]
Great work!  Is it compatible with gptk 1.02?

**Reply by JoshBashed** [reply]
Yes

**Reply by logiota** [reply]
if you want to upgrade to gptk 1.02 you can delete these directories:
~/Library/Application Support/Whisky
~/Library/Application Support/com.isaacmarovitz.Whisky

**Comment by 3dprinting-noob** [comment]
How do I do a complete whiskey uninstall and reinstall?

**Reply by benwaco** [reply]
> How do I do a complete whiskey uninstall and reinstall?
> 
> 

[AppCleaner](https://freemacsoft.net/appcleaner/) is always handy :)

**Reply by JoshBashed** [reply]
```sh
sudo rm -rf /Applications/Whisky.app
rm -rf ~/Library/Application\ Support/com.isaacmarovitz.Whisky
rm -rf ~/Library/Containers/com.isaacmarovitz.Whisky
```

This will delete all bottles, data, and the app itself.

**Reply by 3dprinting-noob** [reply]
ok .. so I did the  above, brew updated gptk to 1.02, then ran the whiskey app-- now getting no graphic card error when trying to open Diablo 4
all was ok with the 0.93 whiskey version....
so what do I do now
